trade-finance-manager-ui/templates/case/deal/deal.njk

```diff
@@ -113,4 +114,16 @@
     </div>
   </div>
 
+  {% if showDealCancelButton %}
+    <div class="govuk-!-padding-top-7">
+      {{ govukButton({
+        text: "Cancel deal",
+        classes: "govuk-button--warning",
+        attributes: {
+          'data-cy': 'cancel-deal-button'
```

Comment:
I think I'd be interested for you to chat to Tony about how he'd approach naming this button in terms of data-cy. We're looking to name these more genericly -- but I wonder if this can be a specific button.

Reply:
I think this shouldnt be confused with a `cancel` link & we dont want to commonise it into that.
I think this is its own unique component & can be accessed from the deal page file

Reply:
I agree

Reply:
@ttbarnes Are you happy with this `data-cy` attribute naming? It's not named generically, but it is important to distinguish this 'cancel deal' button from general 'cancel' links

Reply:
Thank you both, can we call this `cancel-button` please?

Recently changed all "cancel" buttons (amongst other things) to have the same `data-cy`.

This way they all use the same E2E partial and click event command 😎

Reply:
I've renamed to `cancel-button`. Marking as resolved 😃